tests/Tests/500__jupyterhub/image-iteration.robot

```diff
@@ -32,13 +32,15 @@ Iterative Testing Classifiers
   END
 
 Iterative Testing Clustering
-  [Tags]  Sanity  POLARION-ID-Clustering
+  [Tags]  Tier2  POLARION-ID-Clustering
```

Comment:
should be Tier3 for long running tests

Reply:
As agreed in the chat, we'll merge this PR with Tier2 in order to release ods-ci 1.13, but we'll discuss in the automation meeting if Tier3 is more appropriate